SimCalorimetry/EcalSimProducers/interface/EcalTimeDigiProducer_Ph2.h

```diff
@@ -0,0 +1,60 @@
+#ifndef SimCalorimetry_EcalSimProducers_EcalTimeDigiProducer_Ph2_h
+#define SimCalorimetry_EcalSimProducers_EcalTimeDigiProducer_Ph2_h
+
+#include "DataFormats/Math/interface/Error.h"
+
+#include "SimCalorimetry/EcalSimAlgos/interface/EcalDigitizerTraits_Ph2.h"
+#include "SimGeneral/MixingModule/interface/DigiAccumulatorMixMod.h"
+#include "SimDataFormats/CaloHit/interface/PCaloHit.h"
+#include "DataFormats/EcalDigi/interface/EcalDigiCollections.h"
+#include "FWCore/Utilities/interface/EDGetToken.h"
+#include "FWCore/Framework/interface/ProducesCollector.h"
+#include "FWCore/Framework/interface/ConsumesCollector.h"
+
+#include <vector>
+
+class ESDigitizer;
+
+class CaloGeometry;
+class EcalSimParameterMap_Ph2;
+class PileUpEventPrincipal;
+class EcalTimeMapDigitizer;
+
+namespace edm {
+  class Event;
+  class EventSetup;
+  template <typename T>
+  class Handle;
+  class ParameterSet;
+}  // namespace edm
+
+class EcalTimeDigiProducer_Ph2 : public DigiAccumulatorMixMod {
```

Comment:
as far as I can tell, `EcalTimeDigiProducer` is only used for Phase 2
@lgray @pmeridian please confirm
if that's the case, you can just modify `EcalTimeDigiProducer` to take out the EE part, and remove this `EcalTimeDigiProducer_Ph2` entirely

Reply:
@lgray confirmed this to me

Reply:
please remove this class and modify the original class as previously described